### PR TITLE
Add GitHub Actions.

### PR DESCRIPTION
**Pull request by adityaruplaha**
Currently runs on PRs and pushes to `main`.
Builds and lints code.

### CODE REVIEW

**Comment by adityaruplaha** [comment]
Looks good to merge. There's not much in this, just enforcing good code.

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install modules.
+        run: yarn
+      - name: Build 'src'.
+        run: yarn build
+      - name: Build 'electron'.
+        run: yarn build-electron
+  lint:
+    runs-on: ubuntu-latest
```
Comment (flavioislima) [inline .github/workflows/main.yml:22]: This job could run on the previous docker. Also, why do we need to build the app first to run the linting?
I think you could just create the docker machine, run yarn and then run the linting and the prettier as well. :)

Comment (adityaruplaha) [reply .github/workflows/main.yml:22]: Okay I'm gonna split it into 2 different job files entirely then. That is what you want right?

Comment (flavioislima) [reply .github/workflows/main.yml:22]: No. I dont think you need to actions for that. You just need to remove the first one.

Comment (flavioislima) [reply .github/workflows/main.yml:22]: I think this should work:

```
name: Linting

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
  workflow_dispatch:

jobs:
  lint:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Install modules.
        run: yarn
      - name: Lint code.
        run: yarn lint
```

Comment (adityaruplaha) [reply .github/workflows/main.yml:22]: So no need to run build? I thought it would be nice to check for that as well.

**Review by ianwijma — APPROVED** [review]
Actions look good.

**Comment by adityaruplaha** [comment]
@flavioislima have a look then, I'm ready to merge.

**Comment by adityaruplaha** [comment]
Talked to him on Discord, looks good, merging.

**Pull request merged**